components/perf-tools/likwid/SPECS/likwid.spec

```diff
@@ -83,7 +83,14 @@ It consists of:
           FC="ifort" \
           FCFLAGS="-module ./" \
 %else
+    %ifarch aarch64
+          COMPILER="GCCARMv8" \
+          BUILDDAEMON="true" \
```

Comment:
I am re-testing at OBS without the `BUILD**` options. Their values are `true` by default anyway.

Reply:
No. It seems those are needed for ARM64.